tests/components/google/conftest.py

```diff
@@ -156,3 +225,49 @@ def mock_insert_event(
     insert_mock = Mock()
     calendar_resource.return_value.events.return_value.insert = insert_mock
     return insert_mock
+
+
+@pytest.fixture(autouse=True)
+def set_time_zone(hass):
+    """Set the time zone for the tests."""
+    # Set our timezone to CST/Regina so we can check calculations
+    # This keeps UTC-6 all year round
+    hass.config.time_zone = "CST"
```

Comment:
Note to reviewer: In addition to moving `set_time_zone` here, I also added this line which is used by the new test that sets the date/time added in test_calendary.yaml